var/spack/repos/builtin/packages/allinea-forge/package.py

```diff
@@ -4,7 +4,16 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack import *
+import platform
 
+def getDistroName():
+    distro = platform.linux_distribution()
```

Comment:
On OSX:

```python
>>> platform.linux_distribution()
('', '', '')
```

Reply:
this package is only for linux, added conflict